docs/dev/placement-api.md

```diff
@@ -0,0 +1,69 @@
+# Node Placement configuration for Template Validator and Node Labeller
+
+1. The cluster admin can modify the Template Validator/Node Labeller CR to inform what nodes the respective operator pods should be deployed on.
+2. Template Validator pods are placed on nodes that expect infra components according to the HCO API. 
+
+```
+Example TV CR with empty Affinity, Node Selector and Toleration values:
```

Comment:
Please prepend this line with `#`, to make it a comment in a yaml file, same for NL

Reply:
Done